app_python/tests/test_main.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+from msk_time.app import app
+
+
+@pytest.fixture
+def client():
+    with app.test_client() as client:
+        yield client
+
+
+@pytest.mark.parametrize(
+    "path, expected",
+    [("/", 200), ("/not_existing_path", 404)],
```

Comment:
o.O
the more you know

Reply:
is `@pytest.mark.parametrize` named after you?